test/easyconfigs/easyconfigs.py

```diff
@@ -472,7 +472,10 @@ def check_https(self, changed_ecs):
             if any(ec_fn.startswith(x) for x in whitelist):
                 continue
 
-            for http_url in http_regex.findall(ec.rawtxt):
+            # ignore commented out lines in easyconfig files when checking for http:// URLs
+            ec_txt = '\n'.join(l for l in ec.rawtxt.split('\n') if not l.startswith('#'))
```

Comment:
The correct thing is to ignore comments altogether, not just lines that start with #.

Reply:
I think this is good enough, I doubt we'll hit `http://` URLs in inline comments?

Also, stripping off all comments isn't that trivial, since there may appear `#` characters in other places (like in string values)

Reply:
Agree that it is non-trivial and should be left for someday when we have nothing else to do...